### DISCUSSION TITLE
Writing "---" in the patch description messes up the layout/font size.

### DISCUSSION THREAD

**Discussion by franckaubry** [cables-gl/cables_docs#243] [! Welcome - readme first!]
When writing "---" in the patch description, the layout/font size is all messed up.

Edit Description:
![image](https://user-images.githubusercontent.com/97125854/169273130-9fb2b1f4-818f-4f08-aede-b9f634932395.png)

Result Description:
![image](https://user-images.githubusercontent.com/97125854/169272861-fcdcfecd-3e03-4159-838a-793a14b23230.png)

**Comment by steam0r** [comment]
you are able to use markdown here (https://www.markdownguide.org/basic-syntax/) and have basically created headlines (see "Alternate Syntax" on that page).

could you paste the actual text here, so we can at least check the styling. thanks!

**Reply by franckaubry** [reply]
"Styling" is probably what I should have written in the subject.

```
---
UPDATE: The Smooth op (NEW) now behaves as expected on https://dev.cables.gl/p/Gd2J80
---
```

**Reply by steam0r** [reply]
tried to at least make it look less broken (on dev.cables.gl), point with the headlines created from markdown still stands ;)

![Bildschirmfoto 2022-06-01 um 18 49 17](https://user-images.githubusercontent.com/151271/171458060-ee6ced49-87a8-45bd-b68b-2f25de683ba7.png)

**Reply by franckaubry** [reply]
👌
